### DISCUSSION TITLE
How to find network folders

### DISCUSSION THREAD

**Discussion by fr8nki3** [mikiher/audiobookshelf-windows#21] [Q&A] [answered]
I have tried manually entering in the address, but I cannot locate my mapped drives.
I tried looking into the configs but I cannot understand where to make any adjustments, and I was hoping for something a little more user-friendly. Any direction is appreciated thanks.

**Comment by fr8nki3** [comment]
I tried using mkdir and was able to get it to recognize a library, but now it is providing an error:

fileUtils] Failed to getFileTimestampsWithIno for path "C:/AudioBooks/192.168.50.43/Public/Shared Music/AudioBooks/Fiction/Nuttall, Christopher G/The Empire's Corps/Christopher G. Nuttall - The Empire's Corps Book 14 - Wolf's Bane/Wolfs Bane/Wolfs Bane/Wolfs Bane (Unabridged).m4b" Error: ENOENT: no such file or directory, stat 'C:\AudioBooks\192.168.50.43\Public\Shared Music\AudioBooks\Fiction\Nuttall, Christopher G\The Empire's Corps\Christopher G. Nuttall - The Empire's Corps Book 14 - Wolf's Bane\Wolfs Bane\Wolfs Bane\Wolfs Bane (Unabridged).m4b'

**Reply by fr8nki3** [reply]
Here is my log file, shortened for the 25MB limit.
[2024-07-24-edit for size.txt](https://github.com/user-attachments/files/16367981/2024-07-24-edit.for.size.txt)

**Comment by mikiher** [comment]
Hey, sorry for getting back late on this.

What is the folder: C:/AudioBooks/192.168.50.43 ? Is it a symbolic link of some kind?

I have SMB mapped network drives on my system, and they are working well with Audiobookshelf. What type of network FS are you trying to access?

**Comment by mikiher** [comment, marked as answer]
Ah, I think I know what's happening. 

It's essentially the same as the [Feature Request: Support for network paths](https://github.com/mikiher/audiobookshelf-windows/discussions/20). Let's close this one and continue discussion over there.

**Comment by mikiher** [comment]
Closing at dup of #20
